mllib/src/main/scala/org/apache/spark/ml/optim/WeightedLeastSquares.scala

```diff
@@ -94,8 +110,7 @@ private[ml] class WeightedLeastSquares(
       if (standardizeFeatures) {
         lambda *= aVar(j - 2)
       }
-      if (standardizeLabel) {
-        // TODO: handle the case when bStd = 0
+      if (standardizeLabel && bStd != 0) {
```

Comment:
Thanks. As you said, we will expect non zero coefficients in this case, so we don't have to match glmnet. 

However, we may want to throw excpetion when standerizeLabe is true, and ystd is zero since the problem is not well defined. 

Thanks.